babynames.py

```diff
@@ -14,6 +14,9 @@
 import re
 import argparse
 
+
+__author__ = "knmarvel"
```

Comment:
I ended up putting "... and madarp" on mine because he wrote the parsing function that I used.